roles/forkliftcontroller/templates/deployment-ui.yml.j2

```diff
@@ -23,6 +23,10 @@ spec:
           image: "{{ ui_image_fqin }}"
           imagePullPolicy: "{{ image_pull_policy }}"
           env:
+{% if minikube_setup|bool %}
```

Comment:
We already auto-detect if we are running on a k8s/minikube cluster and actually set a variable within operator for these purposes , the name is k8s_cluster, for reference you see here : https://github.com/konveyor/forklift-operator/blob/main/roles/forkliftcontroller/tasks/main.yml#L27 , so I don't think we have a need for a specific minikube_setup variable , seems redundant.

Reply:
I still think that we need to separate the k8s and minikube. AFAIK the k8s is using the AAA by defualt and it would not work with default `AUTH_REQUIRED=false` because it would not have the Bearer tokens for communications with the controller. The minikube can have the AAA but not by default setup. 
I'm more than happy to change all the var names to suggestions by @jortel https://github.com/konveyor/forklift-operator/pull/230#issuecomment-1175344327

Reply:
@mnecas Let's go with var name suggestions by @jortel first , in any case feature_auth_required should be the boolean to check if to set AUTH_REQUIRED=false on the deployments.